README.md

```diff
@@ -7,25 +7,28 @@ NCVX is built on top of [CVXPY](http://www.cvxpy.org/), a domain-specific langua
 
 Installation
 ------------
-You should first install CVXPY version 0.4. CVXPY install guide can be found [here](http://www.cvxpy.org/). We are working on upgrading NCVX to work with CVXPY 1.0. 
+You should first install CVXPY version >= 1.1.13. The CVXPY install guide can be found [here](http://www.cvxpy.org/).
 
-Then install ``scsprox`` from source [here](https://github.com/SteveDiamond/scsprox).
+Then install ``scsprox`` from source [here](https://github.com/bettbra/scsprox).
 
-The easiest way to install the package is to run ``pip install ncvx``. To install the package from source, run ``python setup.py install`` in the main folder. The package has CVXPY and lap as dependencies.
+The easiest way to install the package is to run ``pip install ncvx``. To install the package from source, run ``python setup.py install`` in the main folder.
 
 Example
 -------
 The following code uses NC-ADMM heuristic to approximately solve a least-squares problem where the variable has only ``k`` nonzero components, and all components are between -1 and 1.
 ```
+from cvxpy import sum_squares, sum, Problem, Minimize
```

Comment:
We tend to do ``import cvxpy as cp`` these days. Then you would have ``cp.sum_squares``, etc.

Reply:
I've fixed the README example to have the more standard cvxpy import.